src/lib/InPort.coffee

```diff
@@ -136,33 +136,32 @@ class InPort extends BasePort
       throw new Error 'Contains query is only possible on buffered ports'
     @buffer.filter((packet) -> return true if packet.event is 'data').length
 
-  # Fetches a packet from the port
-  get: (scope) ->
+  getBuffer: (scope) ->
```

Comment:
That is a good question. Could also be internal, but in any case useful since we were duplicating this code in multiple methods